charts/rancher-webhook/1.0.3+up0.2.5/Chart.yaml

```diff
@@ -1,6 +1,7 @@
 annotations:
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/hidden: "true"
+  catalog.cattle.io/kube-version: '>= 1.16.0-0 < 1.25.0-0'
   catalog.cattle.io/namespace: cattle-system
   catalog.cattle.io/os: linux
   catalog.cattle.io/permits-os: linux,windows
```

Comment:
Hmm, I can't figure out where this chart is used. In 2.6.4 we use 1.0.4, and in 2.6.3-patch3 we used `1.0.3+up0.2.4` 
https://github.com/rancher/rancher/blob/v2.6.3-patch3/package/Dockerfile#L49
Either way this one shouldn't have rancher-version of <2.5.99-0, I don't know how that got added. 
Thats not in the upstream: https://github.com/rancher/webhook/blob/v0.2.5/charts/rancher-webhook/Chart.yaml
I think the installer for webhook just ignores this annotation.